crates/krun/src/bin/krun.rs

```diff
@@ -103,7 +103,7 @@ fn main() -> Result<()> {
         }
         debug!(cpuset:?; "sched_setaffinity");
         sched_setaffinity(None, &cpuset).context("Failed to set CPU affinity")?;
-        // Configure the number of vCPUs and the amount of RAM (max 16384 MiB).
+        // Configure the number of vCPUs and the amount of RAM (max 32768 MiB).
```

Comment:
This comment here is wrong if we remove the `guard` in `cli_options`. The user could simply specify more than 32768 MiB.